helpers/reviews.js

```diff
@@ -1,18 +1,354 @@
 const Review = require('@pbnj-xintern/xintern-commons/models/Review')
-const dbExec = require('@pbnj-xintern/xintern-commons/util/db')
-const status = require('@pbnj-xintern/xintern-commons/util/status')
-const dbUrl = process.env.MONGO_URL;
+const Status = require('@pbnj-xintern/xintern-commons/util/status')
+const User = require('@pbnj-xintern/xintern-commons/models/User')
+const Rating = require('@pbnj-xintern/xintern-commons/models/Rating')
+const Comment = require('@pbnj-xintern/xintern-commons/models/Comment')
+const Company = require('@pbnj-xintern/xintern-commons/models/Company')
+const db = require('@pbnj-xintern/xintern-commons/util/db')
+const mongoose = require('mongoose')
+
+const MONGO_URL = process.env.MONGO_URL
+
+//--------------- FUNCTIONS ---------------
+
+//Returns user ID
+const findUserId = async (eventBody) => {
+    try {
+        let foundUser = await db(MONGO_URL, () => {
+            return User.find({
+                _id: eventBody.user_id
+            })
+        })
+        console.log('foundUser:\n', foundUser)
+        return foundUser[0]._id
+    } catch (err) {
+        console.error('user does not exist:\n', err.message)
+    }
+}
+
+//Creates a new Rating obj and saves to db. Returns rating ID
+const createRatingAndSave = async (eventBody) => {
```

Comment:
need its own lambda

Reply:
renamed to `createRating` and remains a helper function